### PR TITLE
Ant layout

### PR DESCRIPTION
**Pull request by hyetigran**
##  What does this PR do
Incorporates ant design for layout, header, sider and content components

##  What are the relevant Trello board stories
https://trello.com/c/NMHTe0sy/186-add-the-ant-to-navigation-and-layout

##  Screenshots (If appropriate)
image

##  Questions (If any):

### CODE REVIEW

**Comment by haywhyze** [comment]
You might need to hide the new side-bar on mobile view.

File: workout-tracker/src/views/UserPage/UserPage.css [hunk 1]
```diff
@@ -1,26 +1,19 @@
-.dashboard-container {
-  display: flex;
-  position: fixed;
-  bottom: 0;
-  height: 94%;
+.content.ant-layout-content {
+  margin: 0 !important;
+  padding: 0 !important;
```
Comment (haywhyze) [inline workout-tracker/src/views/UserPage/UserPage.css]: You might want to remove the `!important` as it is considered bad practice as highlighted in [this article on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

File: workout-tracker/src/views/UserPage/UserPage.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { Fragment } from "react";
```
[the remark this answers is not in the document]
Comment (hyetigran) [reply workout-tracker/src/views/UserPage/UserPage.js]: Replaced Fragment with Layout component but left the import.
Removed.

File: workout-tracker/src/index.css [hunk 3]
```diff
@@ -364,3 +366,8 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+html,
+body {
+  height: 100% !important;
```
[the remark this answers is not in the document]
Comment (hyetigran) [reply workout-tracker/src/index.css]: removed !important

File: workout-tracker/src/views/UserPage/UserPage.css (hunk 1 above)
Comment (hyetigran) [reply workout-tracker/src/views/UserPage/UserPage.css]: I'm aware, however it was previously not overriding default ant styles. It seems to be unnecessary now so removed.

File: workout-tracker/src/views/UserPage/UserPage.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (hyetigran) [reply workout-tracker/src/views/UserPage/UserPage.js]: Sure why not. Replaced with <> in all occurrences in the app.

File: workout-tracker/src/components/Layout/Layout.js [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
-
+import "antd/dist/antd.css";
+import { Layout } from "antd";
 import './Layout.css';
 
-const layout = props => (
+const { Header, Content } = Layout;
+
+const MainLayout = props => (
   <Fragment>
```
[the remark this answers is not in the document]
Comment (hyetigran) [reply workout-tracker/src/components/Layout/Layout.js]: done

File: workout-tracker/src/components/MainNavBar/MainNavBar.css [hunk 5]
```diff
@@ -3,10 +3,10 @@
   padding: 0 1rem;
   display: flex;
   align-items: center;
-  max-width: 950px;
+  /* max-width: 950px; */
```
[the remark this answers is not in the document]
Comment (hyetigran) [reply workout-tracker/src/components/MainNavBar/MainNavBar.css]: deleted

**Pull request merged**